app/views/unattended/provisioning_templates/iPXE/ipxe_intermediate_script.erb

```diff
@@ -0,0 +1,22 @@
+<%#
+kind: iPXE
+model: ProvisioningTemplate
+name: iPXE intermediate script
+snippet: false
+-%>
+
+#!ipxe
```

Comment:
I think the file needs to start with this "shebang" rather than a blank line.

Reply:
👍